pkg/agent/openflow/pipeline.go

```diff
@@ -2414,14 +2416,19 @@ func (f *featureService) serviceEndpointGroup(groupID binding.GroupIDType, withS
 		endpointIP := net.ParseIP(endpoint.IP())
 		portVal := util.PortToUint16(endpointPort)
 		ipProtocol := getIPProtocol(endpointIP)
-
 		if ipProtocol == binding.ProtocolIP {
 			ipVal := binary.BigEndian.Uint32(endpointIP.To4())
-			group = group.Bucket().Weight(100).
-				LoadToRegField(EndpointIPField, ipVal).
-				LoadToRegField(EndpointPortField, uint32(portVal)).
-				ResubmitToTable(resubmitTableID).
-				Done()
+			if exportedSvcGroupID != 0 && f.isLocalServiceClusterIP(endpointIP) {
```

Comment:
No need to check if the endpointIP is local cluster Service IP in openflow layer or not, we can check it proxier, and in openflow layer just leverage the expected groupID in the buckets as long as it is valid?